server/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -141,22 +141,24 @@ private QuestionDefinition insertAddressQuestionDefinition() {
   }
 
   private QuestionDefinition insertCheckboxQuestionDefinition() {
-    return questionService
-        .create(
-            new CheckboxQuestionDefinition(
-                "checkbox",
-                Optional.empty(),
```

Comment:
Here an elsewhere, this param `Optional.empty()` was the `enumeratorId`, and that is an optional field in the `MultiOptionQuestionDefinitionConfig` AutoValue, so it doesn't appear on the right side of the diff when set to empty.